packages/pigeon/lib/src/pigeon_lib.dart

```diff
@@ -449,13 +444,158 @@ class PigeonOptions {
       throw Exception(
         'Unable to deduce package name, and no package name supplied.\n'
         'Add a `dartPackageName` property to your `PigeonOptions` config,\n'
-        'or add --dartPackageName={name_of_package} to your command line pigeon call.',
+        'or add --package_name={name_of_package} to your command line pigeon call.',
       );
     }
     return name;
   }
 }
 
+/// Options used when running the code generator.
+class InternalPigeonOptions {
```

Comment:
Does this need to be in an exported file, or could it be moved to another file that's not fully exported for legacy reasons?

Reply:
Hm, I guess it's used in public API below.

Although if we're doing a breaking change for library clients anyway, maybe we can rethink what's exported and what's not?